EventListener/BugsnagListener.php

```diff
@@ -157,11 +157,14 @@ public static function getSubscribedEvents()
             KernelEvents::EXCEPTION => ['onKernelException', 128],
         ];
 
-        // Added with ConsoleEvents::ERROR in Symfony 3.3 to deprecate ConsoleEvents::EXCEPTION
-        if (class_exists('Symfony\Component\Console\Event\ConsoleErrorEvent')) {
-            $listeners[ConsoleEvents::ERROR] = ['onConsoleError', 128];
-        } else {
-            $listeners[ConsoleEvents::EXCEPTION] = ['onConsoleException', 128];
+        // Added ConsoleEvents in Symfony 2.3
+        if (class_exists('Symfony\Component\Console\ConsoleEvents')) {
+            // Added with ConsoleEvents::ERROR in Symfony 3.3 to deprecate ConsoleEvents::EXCEPTION
+            if (class_exists('Symfony\Component\Console\Event\ConsoleErrorEvent')) {
```

Comment:
Both these classes can be imported. PHP `use` statements don't trigger the autoloader. They purely are a name binding. You can then use `ConsoleErrorEvent::class` in place of the string.